direct/src/tkwidgets/EntryScale.py

```diff
@@ -483,16 +483,27 @@ def setNodePathColor(color, np = nodePath, cb = callback):
         'RGBA Panel')
     # Update menu
     menu = esg.component('menubar').component('EntryScale Group-menu')
+    menubar = vgp.component('menubar')
+    menuName = "EntryScale Group"
     # Some helper functions
     # Clear color
     menu.insert_command(index = 1, label = 'Clear Color',
                         command = lambda np = nodePath: np.clearColor())
+    menubar._menuInfo[menuName][1].append("")
+    menubar._menuInfo[menuName][1][2] = menubar._menuInfo[menuName][1][1]
+    menubar._menuInfo[menuName][1][1] = ""
```

Comment:
Could you explain what this bit of code does?

Reply:
As this was already addressed in irc, here the short answer just to have it documented where it should be.

This fixes a "list index out of range" exception that occurs in thePmwMenuBar code whenever the user hovers over some of the menu entries in the tools.

Reply:
I'm not really happy with seeing this unexplained hack repeated all over the code.  Ideally we'd find out why this is needed and fix it elsewhere, or move it to a central location somewhere where it can be changed and documented easily (and taken out if it's no longer needed).